tests/auxtel/test_make_latiss_calibrations.py

```diff
@@ -51,7 +51,7 @@ async def test_configure(self):
             exp_times_dark = 10
             n_flat = 4
             exp_times_flat = [10, 10, 50, 50]
-            detectors = "(0, 1, 2, 3, 4, 5, 6, 7, 8)"
+            detectors = "[0, 1, 2, 3, 4, 5, 6, 7, 8]"
```

Comment:
Is this supposed to be an array or a string? Also, gain, doesn't make sense to have this in LATISS